errbot/backends/telegram_messenger.py

```diff
@@ -446,18 +504,32 @@ def _is_valid_url(url):
                                            content=content,
                                            msg_type=stream_type,
                                            **meta)
-            log.debug("Requesting upload of {0} to {1} (size hint: {2}, stream type: {3})".format(name,
-                      identifier.username, size, stream_type))
+            log.debug("Requesting upload of {0} to {1} (size hint: {2}, "
+                      "stream type: {3})".format(name, identifier.username,
+                                                      size, stream_type))
 
             stream = content
         else:
             stream = Stream(identifier, content, name, size, stream_type)
-            log.debug("Requesting upload of {0} to {1} (size hint: {2}, stream type: {3})".format(name,
-                      identifier, size, stream_type))
-            self.thread_pool.putRequest(WorkRequest(self._telegram_upload_stream, args=(stream,)))
+            log.debug("Requesting upload of {0} to {1} (size hint: {2}, "
+                      "stream type: {3})".format(name, identifier, size, stream_type))
+            self.thread_pool.putRequest(WorkRequest(self._telegram_upload_stream,
+                                                    args=(stream,)))
 
         return stream
 
+    def callback_message(self, mess):
+        """Processes for commands and dispatches the message to all the plugins."""
+        if self.process_message(mess):
+            # Act only in the backend tells us that this message is OK to broadcast
+            self._dispatch_to_plugins('callback_message', mess)
```

Comment:
It should be handled by https://github.com/errbotio/errbot/blob/master/errbot/core.py#L570 the same way